### PR TITLE
Style improvements for the notification badge icon and the profile switch icon

### PR DESCRIPTION
**Pull request by mstaicu**

### CODE REVIEW

**Comment by mstaicu** [comment]
Here are a few of the screenshots of the changes included in this PR:

screen shot 2018-04-27 at 11 47 53

screen shot 2018-04-27 at 11 52 19

Notification icon is no longer wrapping on the next line on narrow viewport devices.

The profile icon is preserving its grey background even on narrow viewport devices.

File: less/navbar.less [hunk 1]
```diff
@@ -482,6 +480,10 @@
       }
     }
   }
+
+  .badge {
+    background-color: @brand-red;
```
Comment (StevePole) [inline less/navbar.less]: Let's just make all badges red in badges.less

File: less/icons.less [hunk 2]
```diff
@@ -9,7 +9,7 @@
       margin-top: -25px;
       margin-left: -12px;
       min-width: 18px;
-      padding: 2px 3px;
+      padding: 3px 3px 2px 2px;
```
Comment (StevePole) [inline less/icons.less]: Why is this necessary?

Comment (StevePole) [reply less/icons.less]: It doesn't look centered in your screenshot.

Comment (mstaicu) [reply less/icons.less]: I wish we would use two overlaid elements, one which has the actual circle and the one with the actual number for the most precise alignment. We found that padding combination (which nudges the content to the centre) to be the "best" option.

File: less/navbar.less (hunk 1 above)
Comment (mstaicu) [reply less/navbar.less]: Ok, will do that. Didn't want to go that far, so I made it very specific.

**Review by skywritergr — APPROVED** [review]
